lld/test/ELF/gnu-property-align-32.s

```diff
@@ -17,7 +17,9 @@
 # CHECK-NEXT: Info: 0
 # CHECK-NEXT: AddressAlignment: 4
 
-# CHECK:      Note {
+# CHECK:      Size: 0x1C
```

Comment:
I'm not following why you've added the Size check?

Reply:
because when there were Notes at two levels this pinned it to the second level

Reply:
Looks reasonable. This is probably better than checking `}`